test/OpenFeature.Contrib.Providers.Flagd.E2e.RpcTest/OpenFeature.Contrib.Providers.Flagd.E2e.RpcTest.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
I wish there was a way to run the same feature file twice in a single project, but it doesn't seem like there's a way to do that, or to in any way restrict SpecFlow from seeing the duplicate test implementations and failing to bind...

So what I've done is put a base class in one project, and implemented the binding in 2 subclasses, each in a different project.